.travis.yml

```diff
@@ -11,4 +11,4 @@ install:
 script:
 - yarn test
 after_success:
-- yarn coverage
+- yarn run codecov
```

Comment:
Move to circleci

Reply:
Use https://github.com/nuxt-community/module-template/blob/master/template/.circleci/config.yml

Reply:
Could you do that work by any one of project members?
I can't set Codecov's token to CircleCI.

Reply:
We can ask @manniL